package.json

```diff
@@ -51,6 +51,7 @@
     "@whotracksme/reporting": "^7.4.0",
     "apexsankey": "github:ghostery/apexsankey",
     "bowser": "^2.11.0",
+    "file-saver": "^2.0.5",
```

Comment:
This library is very small and not maintained in years. Maybe better to copy the relevant source?